### PR TITLE
add KP3

### PR DESCRIPTION
**Pull request by in-siders17**

### CODE REVIEW

**Comment by tenk28** [comment]
image
All three task for this pull request

File: KP3/SashaSolution/CppSolution/example_1.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <cstdlib>
```
Comment (ILLIA990) [inline KP3/SashaSolution/CppSolution/example_1.cpp:2]: I'm not sure why you're using this library here?
And in next files  the same.
(cstdlib)

File: KP3/SashaSolution/CppSolution/example_3.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    double n, x;
+    double result = 0;
+
+    cout << "Enter n: ";
+    cin >> n;
+
+    cout << "Enter x: ";
+    cin >> x;
+
+    for (double i = 1; i <= n; i++) {
```
Comment (ILLIA990) [inline KP3/SashaSolution/CppSolution/example_3.cpp:25]: In previous files u used ++i, try every where use it

File: KP3/SashaSolution/CppSolution/example_1.cpp (hunk 1 above)
Comment (tenk28) [reply KP3/SashaSolution/CppSolution/example_1.cpp:2]: it's okay, cstdlib used for EXIT_SUCCESS in main return

**Review by tenk28 — CHANGES_REQUESTED** [review]

**Comment by tenk28** [comment]
Good work, merge

**Pull request merged**